assets/js/base/utils/create-notice.ts

```diff
@@ -21,10 +21,10 @@ export const hasStoreNoticesContainer = ( container: string ): boolean => {
 };
 
 const findParentContainer = ( container: string ): string => {
-	if ( container.includes( noticeContexts.CHECKOUT + '/' ) ) {
+	if ( container.includes( noticeContexts.CHECKOUT ) ) {
```

Comment:
Removed the slash do that `wc/cart` errors surface in `wc/checkout` if the `StoreNoticesContainer` is not present.

Reply:
Could you explain this a bit more, not sure I understand fully what the slash did and why removing it is needed

Reply:
The logic was looking for a prefix of `wc/cart/` and then selecting either the parent container, or switching to `wc/checkout`. 

This worked for children of `wc/cart/` but not for `wc/cart` itself. Removing the slash lets it change the container if the `wc/cart` container was passed in.

Reply:
Ahhh right yes because we add errors to inner blocks. Nice one thanks!